index.html

```diff
@@ -0,0 +1,856 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>New York Times Article Page</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+      rel="stylesheet"
+      integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN"
+      crossorigin="anonymous"
+    />
+  </head>
+  <body>
+    <nav>
+      <ul>
+        <li><i class="fa fa-bars" aria-hidden="true"></i></li>
+        <li><i class="fa fa-search" aria-hidden="true"></i></li>
+        <li><a href="#">SPACE &amp; COSMOS</a></li>
+      </ul>
+      <img src="./assets/images/NewYork.PNG" alt="The New YorK Times" />
+      <div class="nav-right">
+        <a href="#">Subscribe now</a>
+        <button>Login</button>
+      </div>
```

Comment:
- The styles are different than the original.
  Original 👇  
  ![Annotation 2020-06-09 222448](https://user-images.githubusercontent.com/57726348/84177246-3b9f3400-aaa0-11ea-8b90-d61553e12eca.jpg)

  Right now 👇  
  ![Annotation 2020-06-09 222437](https://user-images.githubusercontent.com/57726348/84177294-4c4faa00-aaa0-11ea-8937-37ed87a029c1.jpg)